render/topologies.go

```diff
@@ -25,6 +29,45 @@ var ProcessRenderer = MakeReduce(
 	},
 )
 
+// ProcessWithContainerNameRenderer is a Renderer which produces a process
+// graph enriched with container names where appropriate
+type ProcessWithContainerNameRenderer struct{}
+
+// Render produces a process graph where the minor labels contain the
+// container name, if found.
+func (r ProcessWithContainerNameRenderer) Render(rpt report.Report) RenderableNodes {
+	var processes = ProcessRenderer.Render(rpt)
+	var containers = LeafMap{
```

Comment:
[Always prefer type-inferred assignment with :=](https://docs.google.com/document/d/18UTcmy8VoecEaYNb0DrI8NZrI1uJbq-XYDEko0l5Ny4/edit) unless you're doing a var block.